modules/stroeerCoreBidAdapter.js

```diff
@@ -155,10 +153,14 @@ function groupBy(array, keyFns) {
 
 function getVersionValues (win) {
   const pbjs = getGlobal();
-  return {
+  var versionObject = {
```

Comment:
nitpick: `const` is better in this case. you'll still be able to change/add fields on this object.. you just can't change the reference to another object which is what we want.